src/services/jwt.service.ts

```diff
@@ -0,0 +1,33 @@
+import jwt, { Secret } from 'jsonwebtoken';
+
+export class JwtService {
+  private createToken(id: string, secret: string, expiresIn: string) {
```

Comment:
I think better to make this method more universal.
The best way for that - pass object like {key: value}.

type Payload = {
[key:string]: any;
};